server/cpho/jinja2/indicators/_progress_bar.jinja2

```diff
@@ -0,0 +1,86 @@
+
+{% set data_entered_percent = 0 %}
+{% set program_approved_percent = 0 %}
+{% set hso_approved_percent = 0 %}
+{% if submission_statuses['global_status'] in ['not_yet_submitted', 'modified_since_last_submission'] %}
+  {% set data_entered_percent = 100 %}
+{% elif submission_statuses['global_status'] == 'program_submitted' %}
+  {% set data_entered_percent = 100 %}
+  {% set program_approved_percent = 100 %}
+{% elif submission_statuses['global_status'] == 'submission_up_to_date' %}
+  {% set data_entered_percent = 100 %}
+  {% set program_approved_percent = 100 %}
+  {% set hso_approved_percent = 100 %}
+{% endif %}
+
+<div class='row mt-1 gx-2 {% if not progress_in_table %}mb-4{% endif %}'>
+  <div id="{{ indicator.id }}-{{ period.code }}-data-entered"
+       class="col-md-4 col-sm-12">
+    <div class="progress" style="height:0.75rem">
+      <div class="progress-bar bg-warning progress-data-entered"
+           role="progressbar"
+           style="width: {{ data_entered_percent }}%"
+           aria-valuenow="{{ data_entered_percent }}"
+           aria-valuemin="0"
+           aria-valuemax="100"
+           aria-labelledby="{{ indicator.id }}-{{ period.code }}-data-entered"></div>
+    </div>
+    <div class="d-flex justify-content-center {% if not progress_in_table %}mt-1 fw-bold{% endif %}">
+      {{ tdt("Data entered") }}
+      {% if not progress_in_table %}
+        <i tabindex="0"
+           role="tooltip"
+           class="bi bi-info-circle mx-1"
+           data-bs-placement="bottom"
+           data-bs-toggle="tooltip"
+           data-bs-title="{{ tdt("Data currently in database but not approved by program or HSO. Data is not made visible through the API (for infobase) until approved by program and then HSO. Data modified after approval is also treated similarly") }}"></i>
```

Comment:
An [`<i>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i) doesn't seem to make sense here. It indicates "text that is set off from the normal text", which generally just means it's displayed in italics (and a slim chance that some screen readers my read it with an alternate voice). The only text inside this tag is the info symbol character though, added by the `bi-info-circle` class. The symbol itself doesn't have an italicized version, and a screen reader just reads it out as its code "f431" anyway.

The `<i>` itself isn't really a problem, just a weird use of it. The screen reader pronouncing "f431" is more of a problem.

You'll see in my suggested fix that I replaced the `<i>` with two divs. An outer one with the bootstrap tooltip attributes (and an `aria-label`) and an inner one for the icon. I did this so I could mark the icon, and it's "f431" content, as `aria-hidden` since it is just a visual indicator to tell visual users to mouse over it. The equivalent for a screen reader user is an `aria-label` indicating that the element is for accessing "more info", in addition to the `tabindex="0"` which prompts the screen reader to direct the user that there's an interactive element they can tab to within the region it's currently reading.

Reply:
Oh, and [tooltip role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role) is meant to go on the element containing the tooltip content. There's not a dedicated role for the element which triggers a tooltip.